.gitignore

```diff
@@ -68,31 +68,52 @@ instance/
 # Sphinx documentation
 docs/_build/
 
-# PyBuilder
+# Make and PyBuilder
+config/
 target/
 
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# IPython
+profile_default/
+ipyton_config.py
+
 # pyenv
 .python-version
 
-# celery beat schedule file
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
+#   However, in case of collaboration, if having platform-specific dependencies or dependencies
+#   having no cross-platform support, pipenv may install dependencies that don't work, or not
+#   install all needed dependencies.
+Pipfile.lock
+
+# PEP 582; used by e.g. github.com/David-OConnor/pyflow
+__pypackages__/
+
+# Celery stuff
 celerybeat-schedule
+celerybeat.pid
 
 # SageMath parsed files
 *.sage.py
 
 # dotenv
 .env
 
-# virtualenv
+# Environments
+.env
 .venv
+env/
 venv/
 ENV/
+env.bak/
+venv.bak/
```

Comment:
not sure we should gitignore all possible virtualenv names used elsewhere :)

Reply:
It's a common practice for Python https://github.com/github/gitignore/blob/master/Python.gitignore#L107